src/clr_bundle_add.c

```diff
@@ -53,13 +56,19 @@ static void print_help(void)
 }
 
 static const struct option prog_opts[] = {
-	{ "skip-optional", no_argument, &skip_optional_bundles, 1 },
-	{ "skip-diskspace-check", no_argument, &skip_diskspace_check, 1 },
+	{ "skip-optional", no_argument, 0, FLAG_SKIP_OPTIONAL },
+	{ "skip-diskspace-check", no_argument, 0, FLAG_SKIP_DISKSPACE_CHECK },
 };
 
 static bool parse_opt(int opt, UNUSED_PARAM char *optarg)
 {
 	switch (opt) {
+	case FLAG_SKIP_OPTIONAL:
+		skip_optional_bundles = optarg ? strtobool(optarg) : true;
```

Comment:
since these options are local they don't need that logic,`skip_optional_bundles = true` should be enough since we are ignoring values of `false` from the config_loader

Reply:
true